frontend/pages/policies/ManagePoliciesPage/components/PoliciesListWrapper/PoliciesListWrapper.tsx

```diff
@@ -96,29 +97,33 @@ const PoliciesListWrapper = ({
         canAddOrRemovePolicy ? "" : "hide-selection-column"
       }`}
     >
-      <TableContainer
-        resultsTitle={resultsTitle || "policies"}
-        columns={generateTableHeaders({
-          selectedTeamId: selectedTeamData?.id,
-          showSelectionColumn: canAddOrRemovePolicy,
-          tableType,
-        })}
-        data={generateDataSet(policiesList)}
-        isLoading={isLoading}
-        defaultSortHeader={"name"}
-        defaultSortDirection={"asc"}
-        manualSortBy
-        showMarkAllPages={false}
-        isAllPagesSelected={false}
-        disablePagination
-        onPrimarySelectActionClick={onRemovePoliciesClick}
-        primarySelectActionButtonVariant="text-icon"
-        primarySelectActionButtonIcon="delete"
-        primarySelectActionButtonText={"Delete"}
-        emptyComponent={NoPolicies}
-        onQueryChange={noop}
-        disableCount={tableType === "inheritedPolicies"}
-      />
+      {!isLoading ? (
```

Comment:
A nit, but why not: 

```
{isLoading ? (
  <Spinner /> 
) : (
  <TableContainer />
)}
```

Checking for true on `isLoading` makes the code read a bit more naturally.

Reply:
Made an issue to fix the other two examples of this already pushed in the code: https://github.com/fleetdm/fleet/issues/3508

Reply:
Thanks, it's entirely possible I pushed one or both of the other examples :P